cmake/triggerConfig.cmake.in

```diff
@@ -16,6 +16,7 @@ find_dependency(readoutlibs)
 find_dependency(iomanager)
 find_dependency(utilities)
 find_dependency(hdf5libs)
+find_dependency(readoutmodules)
```

Comment:
and appdal?

Reply:
Not sure how I missed this, corrected. Cheers!